src/fund/policies/risk-management/PriceTolerance.sol

```diff
@@ -41,7 +41,9 @@ contract PriceTolerance is TradingSignatures, DSMath {
         view
         returns (bool)
     {
-        IPriceSource pricefeed = IPriceSource(Spoke(msg.sender).priceSource());
+        IPriceSource pricefeed = IPriceSource(
```

Comment:
Helper getter functions for policies?

Reply:
@fubhy Came up with a nice solution here: https://github.com/melonproject/protocol/pull/989/commits/ec508e584ad6e06a33bf9d825171c1c948f502dd

I abstracted routing into a `FundRouterMixin`, which I think is important because it guarantees that routing logic happens in a fund and in plugins in exactly the same way.

We can either use this directly in plugins and pass `ISpoke(msg.sender).HUB()` in as an argument (e.g., `IPriceSource(__getPriceSource(ISpoke(msg.sender).HUB()))`), or create a `SpokeCallee` contract that inherits `FundRouterMixin` and will simplify this to `IPriceSource(__getPriceSource())`, `IRegistry(__getRegistry())`, etc. I think that latter option will be the way to go.

I will implement this in #986 (no sense in doing it to policies and fees now, as this will all change shortly).

Reply:
I think we just need to create getter functions in hub public so that plugin can call these functions via hub. It's quite redundant to have a mixin here because it will add more code to the plugin contract